apps/rxjs.dev/tools/transforms/angular-base-package/index.js

```diff
@@ -37,7 +37,7 @@ module.exports = new Package('angular-base', [
 
   // overrides base packageInfo and returns the one for the 'angular/angular' repo.
   .factory('packageInfo', function () {
-    return require(path.resolve(PROJECT_ROOT, 'package.json'));
+    return require(path.resolve(PROJECT_ROOT, './packages/rxjs/package.json'));
```

Comment:
I think the `PROJECT_ROOT` path should be fixed, not the other way around. Suggestion (after `PROJECT_ROOT` path is fixed):

```suggestion
    return require(path.resolve(PROJECT_ROOT, 'package.json'));
```